less/_components/discover-search.less

```diff
@@ -36,3 +36,22 @@
   }
 }
 
+.featured-jumbotron {
+  .form-search-projects {
```

Comment:
I would prefer that we implement a solution that would allow us to re-use these styles for other sites.

I am thinking that should replace .form-search-projects with .input-group

Also, I don't think that all of these css properties are needed if we remove the .textfield-underline  class from the .form-control.

Reply:
.textfield-underline class is also used here (following image), Do you also wanna change this one? 

![image](https://user-images.githubusercontent.com/39588094/105076000-82063980-5a58-11eb-9826-b57f961d2aa0.png)

Reply:
I noticed that on page: https://accounts.eclipse.org/contact/membership we are also using input-group. Do we want to change them all to border-rounded?